src/components/BarChart/BarChart-container.tsx

```diff
@@ -2,10 +2,12 @@ import React, { useState } from 'react';
 import BarChartView from './BarChart-view';
 import clsx from 'clsx';
 import { BarChartSegment } from './BarChart-styles';
-import { Wallet, Accumulator } from './types';
+import { Wallet, Accumulator, Props } from './types';
 
-const BarChartContainer = ({ width = 2000 }) => {
-  const wallets: Wallet[] = []; //TODO: import data 
+const BarChartContainer = (props: Props) => {
+  const { width, wallets = [] } = props;
+
+  const classes = BarChartSegment();
```

Comment:
since it's a hook, call it with `use` prefix and `styles` suffix like `useBarChartSegmentStyles` or do like in material-ui's examples and call it simply `useStyles`.